src/backfill/prioritizer.ts

```diff
@@ -1,14 +1,15 @@
-import { RepositoryData } from "models/subscription";
```

Comment:
If `looper.ts` is gone, `prioritizer.ts` can go, too. Unless we want to finally refactor the backfilling mechanism

Reply:
we're not using prioritizer at all it seems, I'm glad to remove it since we need to redo the backfill properly.